templates/control

```diff
@@ -0,0 +1,7 @@
+Package: ${project.artifactId}
+Version: ${project.version}
+Section: contrib/utils
+Priority: optional
+Architecture: all
```

Comment:
Should we add a dependency on Java and Git?  The example in the documentation contains:

> Depends: jdk (>= 1.5)